airbrake/notifier.py

```diff
@@ -83,6 +70,30 @@ def __init__(self, project_id=None, api_key=None, environment=None,
                 "by passing in the arguments explicitly."
             )
 
+        environment = config.get("environment",
+                                 os.getenv('AIRBRAKE_ENVIRONMENT'))
```

Comment:
Correct, was just throwing out the idea when I had the opportunity. Thanks for writing up the issue